pages/linux/i3.md

```diff
@@ -0,0 +1,40 @@
+# i3
+
+> A dynamic tiling window manager.
+> More information:<https://i3wm.org/docs/userguide.html>.
+
+- Start i3 (Note you must not have a window manager open when using this command.):
```

Comment:
Generally we try to avoid the 2nd person (e.g. `you`, `your`, etc) in tldr-pages :slightly_smiling_face:
```suggestion
- Start i3 (Note that a pre-existing window manager must not be open when this command is run.):
```